metricbeat/module/vsphere/datastore_usage/_meta/data.json

```diff
@@ -0,0 +1,19 @@
+{
+    "@timestamp":"2016-05-23T08:05:34.853Z",
+    "beat":{
+        "hostname":"beathost",
+        "name":"beathost"
+    },
+    "metricset":{
+        "host":"localhost",
+        "module":"vsphere",
+        "name":"datastore_usage",
+        "rtt":44269
+    },
+    "vsphere":{
+        "datastore_usage":{
+            "example": "datastore_usage"
```

Comment:
Needs update with an example event. You can run `go test -tags=integration github.com/elastic/beats/metricbeat/module/vsphere/... -data` when your environment is running.